README.md

```diff
@@ -67,7 +67,7 @@ Run FediFetcher as a GitHub Action, a cron job, or a container:
    1.  Go to Settings > Secrets and Variables > Actions
    2.  Click New Repository Secret
    3.  Supply the Name `ACCESS_TOKEN` and provide the Token generated above as Secret
-3. Create a file called `config.json` with your [configuration options](#configuration-options) in the repository root. **Do NOT include the Access Token in your `config.json`!**
+3. Create a file called `config.json` with your [configuration options](#configuration-options) in the repository root. **REMOVE the Access Token line from your `config.json`!**
```

Comment:
I'm not sure I like changing the wording to 'remove': The access token should never be included in the `config.json` in the first instance, rather than removing it: Once it's been added to GitHub, removing it does no good: it's already exposed.